scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -193,6 +193,12 @@ abstract class LocalScanner(name: String, config: ScannerConfiguration) : Scanne
     ): List<ScanResult> {
         val scannerDetails = getDetails()
 
+        if (pkg.isMetaDataOnly) {
+            log.info { "Skipping ${pkg.id.toCoordinates()} as it is meta data only." }
```

Comment:
We usually put `${pkg.id.toCoordinates()}` in single quotes.

Reply:
Done.